src/util.c

```diff
@@ -633,3 +634,20 @@ All facets connected.  No further nearby check necessary.\n");
     stl_verify_neighbors(stl);
   }
 }
+
+void stl_write_error_message(stl_file *stl, const char *msg){
+
+  if(stl->error_buffer != NULL)
+    free(stl->error_buffer);
+
+  // Check for library errors and append additional error message if available.
+  if(errno != 0) {
+    const char *delimiter = ", ";
```

Comment:
Before this change:

```
$ ./admesh test/conveyor/basic.stls 
...
Opening test/conveyor/basic.stls
stl_initialize: Couldn't open test/conveyor/basic.stls for reading: No such file or directory
```

After you change:

```
$ ./admesh test/conveyor/basic.stls 
...
Opening test/conveyor/basic.stls
stl_initialize: Couldn't open test/conveyor/basic.stls for reading, No such file or directory
```

Let's keep it unchanged?

```suggestion
    const char *delimiter = ": ";
```